qiskit_ibm_provider/ibm_backend.py

```diff
@@ -959,13 +851,6 @@ def active_jobs(self, limit: int = 10) -> None:
         """Return the unfinished jobs submitted to this backend."""
```

Comment:
This method can be deleted too since it's related to job limit.

Reply:
`active_jobs` just returns all unfinished jobs doesn't seem related to job limit - is it still worth keeping?

```
return self.provider.backend.jobs(status="pending", limit=limit)
```

Reply:
oh I thought it's related to job limit because there's an `active_jobs` field in job limit. I don't think it's worth keeping since people can just do the query.